app/views/samples/upload_results.haml

```diff
@@ -13,14 +13,23 @@
     .col.pe-3
       %label Results
     .col
-      #uploaded-files
-
+      .row.samples-row
+        .col
+          .text-bold#file-count
+            0 files
+        .col.align-right
+          =link_to '/template.csv', target: "_blank" do
```

Comment:
We might want a better name. For example a `templates` directory with a `upload_samples_results.csv` file.